client/my-sites/subscribers/components/subscriber-stats/subscriber-stats.tsx

```diff
@@ -0,0 +1,63 @@
+import { Icon } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { useSubscriberStatsQuery } from '../../queries';
+import { SubscriberStatsCard } from '../subscriber-stats-card';
+import { ChartIcon, MailIcon, SelectIcon } from './icons';
+import './style.scss';
+
+type SubscriberStatsProps = {
+	siteId: number;
+	subscriptionId?: number;
+	userId?: number;
+};
+
+const SubscriberStats = ( { siteId, subscriptionId, userId }: SubscriberStatsProps ) => {
+	const translate = useTranslate();
+
+	const { data: subscriberStats, isLoading } = useSubscriberStatsQuery(
+		siteId,
+		subscriptionId,
+		userId
+	);
+
+	const openRate = useMemo( () => {
+		if ( subscriberStats && subscriberStats.emails_sent ) {
+			return ( ( subscriberStats.unique_opens / subscriberStats.emails_sent ) * 100 ).toFixed();
+		}
+		return 0;
+	}, [ subscriberStats ] );
+
+	const clickRate = useMemo( () => {
+		if ( subscriberStats && subscriberStats.emails_sent ) {
+			return ( ( subscriberStats.unique_clicks / subscriberStats.emails_sent ) * 100 ).toFixed();
+		}
+		return 0;
+	}, [ subscriberStats ] );
```

Comment:
These functions share the same logic, it could be moved to a small helper function to calculate the rate.